wordpress-mobile/WordPress-iOS
# Comment List: Swipe Actions

#### Details:
In this PR we'll implement CommentsViewController's Swipe Actions, allowing the user to quickly Moderate / Trash comments.

Needs Review: @frosty  
James, may i bother you with a review?. Thanks in advance sir!

CC'ing @aerych for the `WPTableViewHandler` minor touch.

By the way: *Thanks again* for the `UITableViewRowAction` tip. This exact same PR would look *awesome* in Swift.


Closes #3685

--

#### Testing Steps
1. Launch WPiOS and log into your account
2. Tap over `My Sites` > `(Any Site)` > `Comments`
3. Swipe over any of the cells to reveal the actions

Tap over `Trash` / Approve`/ `Unapprove` and verify they work as expected.
